core/rendered_connection.ts

```diff
@@ -198,15 +197,12 @@ export class RenderedConnection
       ? inferiorRootBlock
       : superiorRootBlock;
     // Raise it to the top for extra visibility.
-    const selected = common.getSelected() === dynamicRootBlock;
-    if (!selected) dynamicRootBlock.addSelect();
     if (dynamicRootBlock.RTL) {
       offsetX = -offsetX;
     }
     const dx = staticConnection.x + offsetX - dynamicConnection.x;
     const dy = staticConnection.y + offsetY - dynamicConnection.y;
     dynamicRootBlock.moveBy(dx, dy, ['bump']);
-    if (!selected) dynamicRootBlock.removeSelect();
```

Comment:
Verify that these changes still work correctly.

Reply:
See https://github.com/google/blockly/pull/9004/files/b7c6aca41e1ccb85b65bb3c7eb211e484c2ecc70#r2078738690.